broker/consumer.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3.7
+# -*- coding: UTF-8 -*-
+
+"""The ``consumer`` module handles the ingestion of a Kafka alert stream into
+Google Cloud Storage (GCS) and defines default connection settings for
+different Kafka servers.
+
+Usage Example
+-------------
+
+.. code-block:: python
+   :linenos:
+
+   from broker.consumer import GCSKafkaConsumer, DEFAULT_ZTF_CONFIG
+
+   # Define connection configuration using default values as a starting point
+   config = DEFAULT_ZTF_CONFIG.copy()
+   config['sasl.kerberos.keytab'] = '<Path to authentication file>'
+   config['sasl.kerberos.principal'] = '<>'
+   print(config)
+
+   # Create a consumer
+   c = GCSKafkaConsumer(
+       kafka_config=config,
+       bucket_name='my-GCS-bucket-name',
+       kafka_topics='ztf_20200301_programid1',
+       pubsub_topic='my-GCS-PubSub-name',
+       debug=True  # Use debug to run without updating your kafka offset
+   )
+
+   # Ingest alerts one at a time indefinitely
+   c.run()
```

Comment:
How do I check the pub sub topic to see these messages?

Reply:
In GCS select `Pub/Sub` -> `Topics` -> `testing-topic`. Programatically I'm not sure but @cnmazz might know.